pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -810,7 +820,10 @@ def process(metadata_dir, conf, es_index, es, donor_output_jsonl_file, bam_outpu
     read_annotations(annotations, 'aug2015', '../pcawg-operations/data_releases/aug2015/release_aug2015_entry.tsv')
     read_annotations(annotations, 's3_transfer_scheduled', '../s3-transfer-operations/s3-transfer-jobs*/*/*.json')
     read_annotations(annotations, 's3_transfer_completed', '../s3-transfer-operations/s3-transfer-jobs*/completed-jobs/*.json')
-    
+    read_annotations(annotations, 'qc_donor_prioritization', 'qc_donor_prioritization.txt')
```

Comment:
This expects the file 'qc_donor_prioritization.txt'  is at the same directory as the parser script. We need to have it copied to there in the 'run_me.sh' script.